src/agent/src/process_options_unix.cpp

```diff
@@ -0,0 +1,95 @@
+#include <process_options.hpp>
+
+#include <agent.hpp>
+#include <agent_registration.hpp>
+#include <fmt/format.h>
+#include <fmt/ranges.h>
+#include <logger.hpp>
+#include <unix_daemon.hpp>
+
+#include <iostream>
+#include <vector>
+
+void RegisterAgent(const std::string& user,
```

Comment:
We need a `process_options.cpp` file to place common functions like this one to avoid code duplication.

Reply:
Done